src/coverlet.msbuild.tasks/coverlet.msbuild.targets

```diff
@@ -3,6 +3,21 @@
   <UsingTask TaskName="Coverlet.MSbuild.Tasks.InstrumentationTask" AssemblyFile="$(CoverletToolsPath)coverlet.msbuild.tasks.dll"/>
   <UsingTask TaskName="Coverlet.MSbuild.Tasks.CoverageResultTask" AssemblyFile="$(CoverletToolsPath)coverlet.msbuild.tasks.dll"/>
 
+  <Target Name="ReferencedPathMaps" BeforeTargets="CoreCompile" >
```

Comment:
Task needs the `Outputs` attribute with the file path that you're writing. That'll ensure that it's added to the list of files for clean.

See example: https://github.com/dotnet/sdk/blob/e605483699158a9202beb4eb9b4e18717c83a6f9/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.GenerateAssemblyInfo.targets#L141

Reply:
ok, thanks.

Reply:
@clairernovotny what do you think about naming target `CoverletReferencedPathMaps`?

Reply:
Actually also other targets doesn't prefix with `Coverlet` so never mind.

Reply:
I tried with output but it doesn't work so after some research I found an article https://docs.microsoft.com/en-us/archive/msdn-magazine/2009/february/msbuild-best-practices-for-creating-reliable-builds-part-1#id0090093 that describe `FileWrites` item list and seems to works.